js/templates/components/formattedCurrency.html

```diff
@@ -0,0 +1,37 @@
+
+<%
+  const price = ob.price; // required
+
+  // If you only need to format a price and not convert, just provide the same currency code for both
+  // the from and to currencies.
+  const fromCur = ob.fromCur; // required
+  const toCur = ob.toCur; // required
+
+  const toolTipBaseClass = ob.toolTipBaseClass || 'toolTip';
+  const toolTipClass = ob.toolTipClass || '';
+  const toolTipFullClass = `${toolTipBaseClass} ${toolTipClass}`;
+  const iconBaseClass = ob.iconBaseClass || 'ion-alert-circled clrTAlert';
+  const iconClass = ob.iconClass || '';
+  const iconFullClass = `${iconBaseClass} ${iconClass}`;
+
+  const showTooltipOnUnrecognizedCur = ob.showTooltipOnUnrecognizedCur || true;
+
+  const fromCurValidity = ob.currencyMod.getCurrencyValidity(fromCur);
+  const toCurValidity = ob.currencyMod.getCurrencyValidity(toCur);
+
+  let tip;
+
+  if (fromCurValidity === 'UNRECOGNIZED_CURRENCY') {
+    tip = ob.polyT('formattedCurrency.unrecognizedCurrencyTip');
+    invalidPrice = true;
+  }
+%>
+
+<%
+  if (fromCurValidity !== 'UNRECOGNIZED_CURRENCY') {
```

Comment:
It could, but I kind of like the separation of the top part being a bunch required js computations whereas the bottom part is where the markup starts and this code is really just spitting out some markup.